test/test_utilities.jl

```diff
@@ -1,5 +1,12 @@
 @testset "Utilities" begin
 
+    @testset "`solve!` does not return anything" begin
```

Comment:
This isn't a new test; it was just moved around during the problem depot changeover and ended up here